src/infinite-list/index.html

```diff
@@ -1,17 +1,17 @@
 <ul ref="wrapper" class="gb-list__wrapper {isGrid ? 'gb-list__wrapper--grid' : 'gb-list__wrapper--column'}">
-  <li data-is="gb-list-item" class="load first-loader" if="{$infinite.isFetchingBackward}" _consumes="infinite">
-    <gb-infinite-loader label="{$infinite.loaderLabel}"></gb-infinite-loader>
+  <li data-is="gb-list-item" class="load first-loader" if="{props.isFetchingBackward}">
+    <gb-infinite-loader label="{props.loaderLabel}"></gb-infinite-loader>
   </li>
-  <li class="load" if="{$infinite.loadMore && $infinite.prevExists}">
-    <gb-button class="load-prev" on-click="{$infinite.clickPrev}">Load Previous</gb-button>
+  <li class="load" if="{props.loadMore && props.prevExists}">
+    <gb-button class="load-prev" on-click="{props.clickPrev}">Load Previous</gb-button>
   </li>
   <li data-is="gb-list-item" class="gb-list__item" each="{item, i in props.items}">
     <yield/>
   </li>
-  <li data-is="gb-list-item" class="load second-loader" if="{$infinite.isFetchingForward}" _consumes="infinite">
-    <gb-infinite-loader label="{$infinite.loaderLabel}"></gb-infinite-loader>
+  <li data-is="gb-list-item" label="{props.loaderLabel}" class="load second-loader" if="{props.isFetchingForward}">
+    <gb-infinite-loader label="{props.label}"></gb-infinite-loader>
   </li>
-  <li data-is="gb-list-item" class="load" if="{$infinite.loadMore && $infinite.moreExists}" _consumes="infinite">
-    <gb-button class="load-more" on-click="{$infinite.clickMore}">Load More</gb-button>
+  <li data-is="gb-list-item" class="load" if="{props.loadMore && props.moreExists}">
+    <gb-button class="load-more" on-click="{props.onclick}">Load More</gb-button>
```

Comment:
This should preserve the name `props.clickMore`. The parent `li` will also need to get it passed down as props since it is actually a `gb-list-item` (note the `data-is`), just like the previous `li`.